src/routes/maven.rs

```diff
@@ -230,7 +230,7 @@ pub async fn version_file(
             .header("Location", &*selected_file.url)
             .body(""));
     } else if file == format!("{}-{}.jar", &string, &version.version_number) {
-        if let Some(selected_file) = version.files.iter().last() {
+        if let Some(selected_file) = version.files.iter().find(|x| x.primary) {
```

Comment:
Not all versions on modrinth have a primary file. Instead of erroring when the version has no primary file, the behavior should be to return the last file of the version. Could you create a new if statement which would handle this case?

Reply:
Oh, ok. I thought that all versions have a primary one. Give me a moment

Reply:
Done. If you want me to rewrite the thing within a single if - let me know